about.md

```diff
@@ -9,9 +9,9 @@ Our Networks is a conference about the past, present, and future of building our
 
 The event has a [Code of Conduct](/code-of-conduct/) in order to foster an environment we can all be in together. During the event if you are being harassed, notice that someone else is being harassed, or have any other concerns, we ask you to contact an organizer immediately. Those who wish to do so but don’t feel comfortable talking to the organizers in-person can email [coc@ournetworks.ca](mailto:coc@ournetworks.ca).
 
-We would like to acknowledge this sacred land on which Our Networks will take place. Tkaronto (Toronto) has been a site of human activity for 15,000 years. This land is the territory of the Huron-Wendat and Petun First Nations, the Seneca, and most recently, the Mississaugas of the Credit River. The territory was the subject of the Dish with One Spoon Wampum Belt Covenant, an agreement between the Iroquois Confederacy and Confederacy of the Ojibwe and allied nations to peaceably share and care for the resources around the Great Lakes. Today, the meeting place of Toronto is still the home to many Indigenous people from across Turtle Island and we are grateful to have the opportunity to work in the community, on this territory.
+We would like to acknowledge this sacred land on which Our Networks will take place. Vancouver is the unceded and ancestral territory of the <span class="font-bc-sans">hən̓q̓əmin̓əm̓</span> and <span class="font-bc-sans">Sḵwx̱wú7mesh</span> (Squamish) speaking peoples, the <span class="font-bc-sans">xʷməθkʷəy̓əm</span> (Musqueam), <span class="font-bc-sans">Sḵwx̱wú7mesh</span>, and <span class="font-bc-sans">səlilwətaɬ</span> (Tsleil-Waututh) Nations, and has been stewarded by them since time immemorial.
```

Comment:
```suggestion
We would like to acknowledge the land on which Our Networks will take place. Vancouver is the unceded and ancestral territory of the <span class="font-bc-sans">hən̓q̓əmin̓əm̓</span> and <span class="font-bc-sans">Sḵwx̱wú7mesh</span> (Squamish) speaking peoples, the <span class="font-bc-sans">xʷməθkʷəy̓əm</span> (Musqueam), <span class="font-bc-sans">Sḵwx̱wú7mesh</span>, and <span class="font-bc-sans">səlilwətaɬ</span> (Tsleil-Waututh) Nations, and has been stewarded by them since time immemorial.
```

Reply:
Should we also maybe have a preamble with some context on event location past? : 

"In previous years we have hosted this event in Tkaronto (Toronto), in territory subject to the Dish with One Spoon Wampum Belt Covenant and the land of the Huron-Wendat and Petun First Nations, the Seneca, and most recently, the Mississaugas of the Credit River.  

After two years online we will be hosting the event in so-called Vancouver, British Columbia. ..."

Reply:
Added in [59696a4](https://github.com/ournetworks/ournetworks.ca/pull/287/commits/59696a4c0ee178f51663976565f5e5b4ddc6cae5).